From 019ec8d706588a488c5f7303d122bc748c0169c8 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Thu, 15 Aug 2013 19:44:59 +0000
Subject: [PATCH 0467/1076] gppdev.idl V1.0

textidl.lib
V1.0.0.8

Mantis 0028544: GPP: Volke/Rickert/H&W-Printer sends initially MEDIA_ABSENT when status is unknown
added mediaStatusType.MEDIA_UNKNOWN




git-svn-id: svn://localhost/SelfServiceCommon/trunk@570 90b65887-3827-0410-9a23-83215b262276
---
 .../Massai/cpp/textIDL/src/local_def.cpp           |  7 ++++++-
 .../Massai/cpp/textIDL/src/textgppdev.cpp          |  1 +
 SelfServiceCommon/Massai/idl/Massai/gppdev.idl     | 14 +++++++++++---
 3 files changed, 18 insertions(+), 4 deletions(-)

diff --git a/SelfServiceCommon/Massai/cpp/textIDL/src/local_def.cpp b/SelfServiceCommon/Massai/cpp/textIDL/src/local_def.cpp
index 16a6fec2..46695995 100644
--- a/SelfServiceCommon/Massai/cpp/textIDL/src/local_def.cpp
+++ b/SelfServiceCommon/Massai/cpp/textIDL/src/local_def.cpp
@@ -1,9 +1,14 @@
 #include "versionlib.h"
 
-VERSION_ID_DT(textIDL, 1, 0, 0, 7);
+VERSION_ID_DT(textIDL, 1, 0, 0, 8);
 
 /*! \page textidl textidl - Release Notes
 
+\date 15.08.2013
+\li V1.0.0.8
+\li Mantis 0028544: GPP: Volke/Rickert/H&W-Printer sends initially MEDIA_ABSENT when status is unknown
+added mediaStatusType.MEDIA_UNKNOWN
+
 
 \date 05.07.2017
 \li V1.0.0.7
diff --git a/SelfServiceCommon/Massai/cpp/textIDL/src/textgppdev.cpp b/SelfServiceCommon/Massai/cpp/textIDL/src/textgppdev.cpp
index cde19541..33f8b2f6 100644
--- a/SelfServiceCommon/Massai/cpp/textIDL/src/textgppdev.cpp
+++ b/SelfServiceCommon/Massai/cpp/textIDL/src/textgppdev.cpp
@@ -66,6 +66,7 @@ namespace MASSAI
         case gppdev::MEDIA_LOW:     return "MEDIA_LOW";
         case gppdev::MEDIA_ABSENT:  return "MEDIA_ABSENT";
         case gppdev::MEDIA_PRESENT: return "MEDIA_PRESENT";
+        case gppdev::MEDIA_UNKNOWN: return "MEDIA_UNKNOWN";
         default:                    return unknown("mediaStatusType", code);
       }
     }
diff --git a/SelfServiceCommon/Massai/idl/Massai/gppdev.idl b/SelfServiceCommon/Massai/idl/Massai/gppdev.idl
index ef967815..980b6fdf 100644
--- a/SelfServiceCommon/Massai/idl/Massai/gppdev.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/gppdev.idl
@@ -22,9 +22,16 @@
  *
  *  @author <b><a href="mailto:carsten.metzler@materna.de">Carsten Metzler (CMe), Materna Information & Communications</a></b>
  *
- *  @version 0.1
+ *  @version 1.0
  * <P>
  *  Real component interface defintion for a general purpose printer.
+ 
+History: 
+15.08.2013
+V1.0 
+Mantis 0028544: GPP: Volke/Rickert/H&W-Printer sends initially MEDIA_ABSENT when status is unknown
+added mediaStatusType.MEDIA_UNKNOWN
+
  */
 
 module gppdev
@@ -82,9 +89,10 @@ module gppdev
 
     enum mediaStatusType
     {
-      MEDIA_LOW,  /**< Media status is low (optional) */
+      MEDIA_LOW,                /**< Media status is low (optional) */
       MEDIA_ABSENT,             /**< Media present in device */
-      MEDIA_PRESENT             /**< Media not present in device */
+      MEDIA_PRESENT,            /**< Media not present in device */
+      MEDIA_UNKNOWN             /**< status of media is unknown */
     };
 
 
-- 
2.41.0.windows.1

